provider/pkg/provider/policy.go

```diff
@@ -45,7 +45,7 @@ type Policy struct {
 	pulumi.ResourceState
 
 	// The policy's ID.
-	ID pulumi.IDOutput `pulumi:"id"`
+	ID pulumi.IDOutput `pulumi:"policyId"`
```

Comment:
I think we'll want to allow `id` for components. I'd hold off on this change until we've addressed the schema validation to allow it.

Reply:
interesting, is there an issue I could read more about this?